tests/Functional/IssuesCest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\Functional;
+
+use App\Tests\FunctionalTester;
+use App\Entity\User;
+use Doctrine\DBAL\Connection;
+
+final class IssuesCest
+{
+    /**
+     * @see https://github.com/Codeception/module-symfony/pull/129
+     * @prepare setClean
+     */
+    public function keepDoctrineDbalConnection(FunctionalTester $I)
+    {
+        $I->haveInRepository(
+            User::class,
+            [
+                'email' => 'fixture@fixture.test',
+                'password' => uniqid(),
+            ]
+        );
+        $ormConnection = $I->grabService('doctrine.orm.default_entity_manager')->getConnection();
+        $I->rebootClientKernel();
+        /** @var Connection $dbalConnection */
+        $dbalConnection = $I->grabService('doctrine.dbal.default_connection');
+
+        $I->assertSame($ormConnection, $dbalConnection);
+
+        $user = $dbalConnection->fetchOne('SELECT id FROM user WHERE email = :email', [':email' => 'fixture@fixture.test']);
+        $I->assertNotFalse($user);
+    }
+
+    protected function setClean(\Codeception\Module\Doctrine2 $doctrine)
+    {
+        $doctrine->_reconfigure(['cleanup' => true]);
```

Comment:
Why do you need this?
The default config of this project [already includes](https://github.com/Codeception/symfony-module-tests/blob/0bbe3972db3d5d7ed61d72574806adf1d4744239/tests/Functional.suite.yml#L12) `cleanup: true`, right?

Reply:
In case the default config is changed once ) But really not necessary atm.

Reply:
let's keep it simple then :-)